Managers/FileManager.hpp

```diff
@@ -0,0 +1,41 @@
+#ifndef SPIC_GAME_FILEMANAGER_HPP
```

Comment:
Controleer deze even overal goed, want sommige hebben `SPIC_SPIC` of `SPIC_GAME`